.github/workflows/blackduck.yml

```diff
@@ -0,0 +1,45 @@
+name: blackduck-scan
+
+on:
+  pull_request: ~
+  push:
+    branches: [2.0]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - run: git fetch --depth=1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ matrix.os }}-${{ hashFiles('**/yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+      - run: sudo apt-get install jq
+      - run: echo "project_version=$(lerna ls -l --json | jq '.[0].version' | tr -d '"')-${GITHUB_REF_NAME}" >> $GITHUB_ENV
+      - name: Blackduck Scan
+        uses: SAP/project-piper-action@master
```

Comment:
I opened an issue in the piper repo to ask them to rename their default branch so we can avoid the "In Solidarity / Inclusive Language" finding for this line.

Feel free to upvote 😄 https://github.com/SAP/project-piper-action/issues/119

Reply:
FYI @fwilhe recommended to use tag 1.0 or branch v1 as a workaround until the default branch is renamed.

Reply:
Is the delta between master and v1 small enough that it doesn't matter that much?

Reply:
`releases/v1` is 1 commit ahead and 68 commits behind. Last commit is 22.9.2020 and on `master` it's 19.7.2021... so I can't say for sure...

Reply:
> Is the delta between master and v1 small enough that it doesn't matter that much?

As far as I can tell (cf commit history) the changes in `master` only relate to dependency updates and rewrite to use the library that was newly created. There might have been a breaking change in the config format in that "use the lib" rewrite, I'm not sure about that, but functionally `v1` should be equivalent.

I'd be happy to help with adaptations if required, but I lost commit access to the repo so I can't just push a `main` branch myself anymore.

Reply:
Another option of course would be to reference a commit id in your workflow which anyway probably is a good idea to ensure the version you use does not change unnoticed.

Reply:
Alright, I've switched to the commit, that should resolve it 👍